src/com/company/model/CreateColumnDefinitionProvider.java

```diff
@@ -0,0 +1,27 @@
+package com.company.model;
+
+import com.company.view.ConsoleReader;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by yulia on 21.02.17.
+ */
+public class CreateColumnDefinitionProvider {
+    public List getProperties(Integer columnNumber) {
+        java.util.List<CreateColumnDefinition> columnDefinitions = new ArrayList<>();
+        for (int i = 0; i < columnNumber; i++) {
+            System.out.println("Now enter column's name and type of this column: <name/type>");
+            ConsoleReader consoleReader = new ConsoleReader();
+            String[] inputParts = consoleReader.read().split("/");
```

Comment:
**org.apache.commons.lang3.StringUtils** provides a lot of useful methods like substringBefore(...) and substringAfter(...)